DOCS-10828 teams (June 12) (#136)
mulesoft/docs-exchange: modules/ROOT/pages/exchange1-faq.adoc

+
. Click Save Changes after editing terms, or Discard to quit any changes you made.

> Note: When you create a category, you must be in the root organization. If you create a category in a business group, the category does not appear.

Reviewer: I think we either use the NOTE notice format or incorporate it into the sentence:
```suggestion
Note that when you create a category, you must be in the root organization. If you create a category in a business group, the category does not appear.
```
Does not appear where?

Author: Fixed.

> == Can I publish whitepapers, Word docs, etc. that aren’t Mule projects or Connectors?

Reviewer: ```suggestion
== Can I publish white papers, Word docs, and so on that aren’t Mule projects or Anypoint Connectors?
```

Author: Fixed.

== How can users view the content of a business group?

> Users must be assigned to the Exchange Viewer role. Users can view the root organization without being assigned to the Exchange Viewer role, but not business groups. If users are invited to a business group, they won't see other business groups until assigned Roles access.

Reviewer: The question is about the business group, so I'd omit the distraction of root access:
```suggestion
Users must be assigned to the Exchange Viewer role to view business groups. If users are invited to a business group, they won't see other business groups until assigned Roles access.
```
Is this question and answer specific to users viewing their own group or other groups? What is "Roles" access? Kinda' confusing in the context of the Exchange Viewer role....

Author: Yes, this page is all legacy content about Exchange 1, which is in the end-of-life process, and uses different systems for access. Fixed.
